jenkins/pyinstaller/v2/requirements.txt

```diff
@@ -6,3 +6,4 @@ dataclasses==0.7
 python-consul==1.1.0
 pathlib==1.0.1
 pytest==5.4.2
+deepdiff==5.5.0
```

Comment:
I think we cannot simply add a third party tool, there is an approval process.
But lets check if we absolutely need this third party tool, we should try to achieve with currently available tools.

Reply:
@Madhura-08 Lets not use any new python package. Please check if a package can be used from https://github.com/Seagate/cortx-re/blob/main/scripts/third-party-rpm/python-requirements.txt. If not, then we should write our own code.

Reply:
Sure. Modified the logic